dropbear.yaml

```diff
@@ -35,9 +35,6 @@ pipeline:
         PROGRAMS="dropbear dropbearkey dbclient dropbearconvert scp"
 
   - uses: autoconf/make-install
-    with:
-      opts: |
-        PROGRAMS="dropbear dropbearkey dbclient dropbearconvert scp"
```

Comment:
FYI @joshrwolf

Reply:
I want to fix this the other way around. And change make install to support args just like make pipeline does.